source/nodejs/adaptivecards/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
 	"extends": "../tsconfig.json",
 	"compilerOptions": {
-		"outDir": "./lib"
+        "outDir": "./lib",
```

Comment:
>        [](start = 1, length = 7)

are we doing spaces or tabs?

Reply:
VS Code is configured for spaces.

Reply:
We're supposed to use tabs for json files generally according to our `.editorconfig`. You can configure VS Code to use the editor config with [this plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig).

Regardless, we shouldn't mix tabs and spaces :)

---
In reply to: [331169850](https://github.com/microsoft/AdaptiveCards/pull/3508#discussion_r331169850) [](ancestors = 331169850)

Reply:
Last time I used tabs in the code I was asked to switch to spaces. Now that I have spaces, I should use tabs :)

I don't know how to configure VS Code to use tabs or spaces depending on the file type, and I also do not know about your guidelines...

Reply:
Our guidelines are encoded in our `.editorconfig`:

https://github.com/microsoft/AdaptiveCards/blob/16172639b4aa70c7461e7b2b52824ae6d5c68d25/.editorconfig#L3-L14

If you configure the plugin, it will do The Right Thing.

Reply:
Ah I guess the plugin you pointed me to is the way to configure make VS Code use different settings for different file types. I will consider that in the future if you can point to what the per-file type guidelines are.

Reply:
Sorry, I posted my last comment before I saw yours. I'll look into it.

Reply:
All you have to do is install the plugin. When you open a file, it will search parent directories until it finds a `.editorconfig` file, which it loads and uses to configure itself. We already have a `.editorconfig` in place.

Reply:
I just did it, and I can see it works.